bin/dormant_user_removal_accuracy_report.py

```diff
@@ -0,0 +1,54 @@
+from datetime import datetime, timedelta
+from services.github_service import GithubService
+from services.slack_service import SlackService
+from config.constants import MINISTRY_OF_JUSTICE
+from utils.environment import EnvironmentVariables
+
+
+def main():
+    required_env_vars = [
+        "GH_MOJ_ADMIN_TOKEN",
+        "GH_AP_ADMIN_TOKEN",
+        "ADMIN_SLACK_TOKEN",
+        "GH_GITHUB_TOKEN_AUDIT_LOG_TOKEN"
+    ]
+
+    env = EnvironmentVariables(required_env_vars)
+
+    operations_engineering_users = [
+        "andyrogers1973",
+        "jasonBirchall",
+        "tamsinforbes",
+        "AntonyBishop",
+        "connormaglynn",
+        "PepperMoJ",
+        "levgorbunov1",
+    ]
+
+    moj_github_org_service = GithubService(env.get(
+        "GH_GITHUB_TOKEN_AUDIT_LOG_TOKEN"), MINISTRY_OF_JUSTICE)
+
+    today = datetime.now()
+    one_week_ago = (today - timedelta(days=7)).strftime('%Y-%m-%d')
+
+    all_removed_usernames = []
+
+    for github_username in operations_engineering_users:
+        removed_users_entries = moj_github_org_service.get_user_removal_events(one_week_ago, github_username)
+        removed_usernames = [entry['userLogin'].lower() for entry in removed_users_entries]
+        all_removed_usernames.extend(removed_usernames)
+
+    current_members = [username.lower() for username in moj_github_org_service.get_org_members_login_names()]
+
+    users_rejoined = [username for username in all_removed_usernames if username in current_members]
```

Comment:
this might be more efficient with sets 
set_c = set_a.intersection(set_b)
but if it doesn't take long to run anyway it's not a big deal